bots/image-download

```diff
@@ -139,14 +139,16 @@ def download(link, force, quiet, stores):
     if not quiet:
         cmd.remove("--silent")
         cmd.insert(1, "--progress-bar")
+    cmd.append("--output")
+    cmd.append(temp)
```

Comment:
I think for this you should close `fd` *before* calling the process, otherwise you have the file open for writing twice (once here and once from curl).